backend/app/infrastructure/routes/user.py

```diff
@@ -152,6 +171,54 @@ def patch(self, user_address: str):
         return {"status": patron_mode_status}, 200
 
 
+@ns.route("/<string:user_address>/antisybil-status")
+@ns.doc(
+    params={
+        "user_address": "User ethereum address in hexadecimal format (case-insensitive, prefixed with 0x)"
+    }
+)
+class AntisybilStatus(OctantResource):
+    @ns.doc(
+        description="Returns user's antisybil status.",
+    )
+    @ns.marshal_with(user_antisybil_status_model)
+    @ns.response(200, "User's cached antisybil status retrieved")
+    def get(self, user_address: str):
+        app.logger.debug(f"Getting user {user_address} cached antisybil status")
+        try:
+            antisybil_status = get_user_antisybil_status(user_address)
+        except UserNotFound:
+            app.logger.debug(
+                f"User {user_address} antisybil status: except UserNotFound"
+            )
+            return {"status": "Unknown"}, 404
+
+        app.logger.debug(f"User {user_address} antisybil status: {antisybil_status}")
+        if antisybil_status is None:
+            return {"status": "Unknown"}, 404
+        score, expires_at = antisybil_status
+        return {
+            "status": "Known",
+            "score": score,
+            "expires_at": int(expires_at.timestamp()),
+        }, 200
+
+    @ns.doc(
+        description="Refresh cached antisybil status",
+    )
+    @ns.marshal_with(user_antisybil_status_model)
+    @ns.response(204, "Refresh succesful")
+    @ns.response(504, "Could not refresh antisybil status. Upstream is unavailable.")
+    def put(self, user_address: str):
+        app.logger.info(f"Updating user {user_address} antisybil status")
+        score, expires_at = update_user_antisybil_status(user_address)
+        app.logger.info(
+            f"User {user_address} antisybil status refreshed {[score, expires_at]}"
+        )
+
+        return None, 204
```

Comment:
should it return None or empty dict?

Reply:
204 is No Content. None would stand for "unknown content", while empty dict, empty string, could be a stand-in for No Content. Very academic distinction.